spec/ParseUser.spec.js

```diff
@@ -1789,7 +1789,7 @@ describe('Parse.User testing', () => {
     });
   });
 
-  it('should allow login with old authData token', done => {
+  it('should not allow login with old authData token', async () => {
```

Comment:
Maybe it should be mentioned in the breaking changes?

Reply:
Yes, this change is related to the opened security advisory.
Normally on login developers should always ask to the provider for new valid auth data, but yes we should mention this in the breaking change with a link to the security advisory.

Reply:
@Moumouls, again I had to redact, just stumbled upon it by chance.